### PR TITLE
Project-1: List Structure

### PR DESCRIPTION
**Pull request by DanSam5K**
# In this Milestone: List structure
 > I was able to complete the following:
- Set up .github/workflows linters to check and fix  linters errors for HTML/CSS/JavaScript.
- Set up the project with webpack that is based on the webpack exercise completed.
- Create an index.html file and write your HTML markup, using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). That can be dynamically displayed with  
     - completed [bool]
     - description [string]
     - index: [number]
- Create a style.css and set rules for the To-Do List, loaded by Webpack Style/CSS Loader.
- All your source files (index.html, index.js and style.css) are located in the/src directory and distribution files are generated and served by webpack.
- Fix all linters errors

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const completedButton = document.createElement('button');
+  completedButton.innerHTML = '<i class="fas fa-check"></i>';
+  completedButton.classList.add('complete-btn');
+
+  todoDiv.appendChild(completedButton);
+
+  // Create Li
+  const newTodo = document.createElement('li');
+  newTodo.innerText = 'Hard coded To do Items';
+  newTodo.classList.add('todo-item');
```
Comment (dasileker) [inline src/index.js:86]: - [ ] You should use a populated  class to add the tasks and it should take the input as argument

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      check: '<i class="fas fa-check"></i>',
+      description: 'Hard Coded To Do',
+      delete: '<i class="fas fa-trash"></i>',
+    },
+  ];
+
+  const todoList = storedTodo;
+
+  todoList.forEach(() => addTodo());
+};
```
Comment (dasileker) [inline src/index.js]: - [x] This should have a description and complete and an index, and also i should see the task without clicking the enter button.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @DanSam5K,** 

### **To Highlight**

- Linters are passing.
- the PR is good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (DanSam5K) [reply src/index.js:86]: ## @dasileker thank you for the feedback.

File: src/index.js (hunk 2 above)
Comment (DanSam5K) [reply src/index.js]: ## Noted. 🤝

**Review by de-scholar — APPROVED** [review]
Hi @DanSam5K  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DanSam5K** [comment]
## @de-scholar  and @dasileker Thank you 🤝 
- Your reviews were motivating and enlightening 🥇 👏

**Pull request merged**